Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -135,3 +135,17 @@ public struct ChannelListQuery: Encodable {
         try pagination.encode(to: encoder)
     }
 }
+
+extension ChannelListQuery {
+    var queryHash: String {
```

Comment:
Is there any reason why we are not using Hashable for that? Do we still want to keep it human readable? If that's the case, is the queryHash format documented somewhere?

Reply:
Hashable uses a random seed while we need it to be stable across app launches so we can determine if there is a query in the database already for the given combination of filter + sorting params. We decided that we should do real hashing so it's a one-way operation.

I'll get back to this topic when reconnection part is done and update the PR so uses some hashing algo instead of taking string representation as a query hash.